tests/installation/language_keyboard/switch_keyboard_layout.pm

```diff
@@ -8,8 +8,14 @@ use base 'y2_installbase';
 use strict;
 use warnings;
 use Test::Assert ':all';
+use testapi;
+use Utils::Backends;
 
 sub run {
+    if (!is_qemu) {
```

Comment:
`unless (is_qemu)` usually nicer (specially when there is single condition, but not big preference.

Reply:
changed.